scripts/all-tests.sh

```diff
@@ -95,13 +95,13 @@ END
     "unit")
         case "$SHARD" in
             "lms")
-                paver test_system -s lms --extra_args="--with-flaky"
+                paver test_system -s lms --extra_args="--with-flaky" || { EXIT=1; }
                 paver coverage
                 ;;
             "cms-js-commonlib")
-                paver test_system -s cms --extra_args="--with-flaky"
-                paver test_js --coverage --skip_clean
-                paver test_lib --skip_clean --extra_args="--with-flaky"
+                paver test_system -s cms --extra_args="--with-flaky" || { EXIT=1; }
```

Comment:
While working on this, I noticed that in the case if one of these fails, the rest do not get run.  This change addresses that.